resources/public/api/conf/1.0/application.yaml

```diff
@@ -34,26 +37,39 @@ paths:
                 allOf:
                   - $ref: '#/components/schemas/nationalInsuranceTestSupportSaAnnualSummaryResponse'
               examples:
-                example-1:
-                  description: Example response for scenario HAPPY_PATH_1
-                  value:
-                    class1:
-                      totalNICableEarnings: 10
-                    class2:
-                      totalDue: 20
-                    maxNICsReached: false
-                example-2:
-                  description: Example response for scenario HAPPY_PATH_2
-                  value:
-                    class1:
-                      totalNICableEarnings: 17
-                    class2:
-                      totalDue: 319.57
-                    maxNICsReached: true
+                happy-path-1:
+                  $ref: '#/components/examples/responseHappyPath1'
+                happy-path-2:

```

Comment:
there is no happy path 2 for NI

Reply:
There is. See [here](https://github.com/hmrc/national-insurance-des-stub/blob/main/resources/public/scenarios/HAPPY_PATH_2.json) and the screenshot. 

<img width="1568" alt="happy path 2" src="https://github.com/hmrc/national-insurance-des-stub/assets/63714619/6ab93eec-df56-4909-8deb-0369daab0485">